allure-generator/src/main/javascript/data/tree/TreeCollection.js

```diff
@@ -69,4 +74,54 @@ export default class TreeCollection extends Collection {
             return this.testcases[index - 1];
         }
     }
+
+    calculateStatistic(items) {
+        const statistic = {};
+        values.forEach(value => {
+            statistic[value] = 0;
+        });
+        items.forEach(item => {
+            if (item.children) {
```

Comment:
you can also add  a check for `item.statistic` and don't do it on every `forEach` loop